jennchenn/uwexplore
# Fix CEAB past course add/delete not updating bars

## Implementation description

- Fixes BREAD-165, BREAD-164
- accidentally broke the live updating when adding/deleting past courses; fixes BREAD-164
- changed calendar tray to only show scrollbar when necessary; fixes BREAD-166
- also added code for refreshing page when clicking on logo (thanks Justin !!)



## Steps to test

1. Click on logo (notice refresh !)
2. Add / delete past courses via the course cards -> verify the CEAB progress bar updates, and state changes are reflected appropriately in past courses container



## What should reviewers focus on?

-
